index.js

```diff
@@ -339,7 +340,8 @@ function joinRoom (room) {
         let doc = node.value
         if (doc.username && doc.publicKey) {
           usernames[doc.publicKey] = doc.username
```

Comment:
we should just store the whole doc so that we can pull all these properties off later.